src/legacy/components/Receive.tsx

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable */
```

Comment:
Since there are so many changes in this legacy file, would you please move it to the `features` folders, so it's not legacy anymore? There is already a lot of work done on it, and it's not really a legacy anymore, except it is still a `class` component and its state and props are untyped.

Reply:
Those tasks are in urgent required by Jeff so that why I worked on weekend so we can include those fixes in the new release. You can add new tasks separately about moving and refactoring in Freedcamp. Let's just review and test this to move forward for the release.